tests/npm-app/src/types/ccf.ts

```diff
@@ -37,6 +37,7 @@ export interface KVMap {
     get: (key: ArrayBuffer) => ArrayBuffer | undefined
     set: (key: ArrayBuffer, value: ArrayBuffer) => KVMap
     delete: (key: ArrayBuffer) => boolean
+    forEach: (callback: (value: ArrayBuffer, key: ArrayBuffer, table: KVMap) => void) => void
```

Comment:
`ccf.ts` exists in two more places currently. Can you update it there as well? After the forum sample fully replaces the tsoa test app, there will only be two in total.